includes/learning-mode/class-sensei-learning-mode-compat.php

```diff
@@ -58,9 +58,9 @@ public function load_theme() {
 	 *
 	 * @access private
 	 */
-	public function the_course_theme_layout() {
+	public function the_learning_mode_layout() {
```

Comment:
When renaming a public function, we should also add an 'alias' to it in the compatibility class:

```php
class Sensei_Course_Theme_Compat extends Sensei_Learning_Mode_Compat {
    public function the_course_theme_layout() { return $this->the_learning_mode_layout() };
}
```

Though we can probably skip the ones marked `@access private`?